packages/ckeditor5-image/src/image/utils.js

```diff
@@ -47,7 +47,7 @@ export function createImageViewElement( writer, imageType ) {
  * @param {'imageBlock'|'imageInline'} matchImageType The type of created image.
  * @returns {module:engine/view/matcher~MatcherPattern}
  */
-export function getImageTypeMatcher( editor, matchImageType ) {
+export function getImgViewElementMatcher( editor, matchImageType ) {
```

Comment:
Is updating the comment in lines 68-69 worth doing?

Reply:
I don't think so. `<picture>` (just like any other wrapper) is completely transparent to this logic.